pyproject.toml

```diff
@@ -70,7 +70,22 @@ exclude = [
 ]
 
 # Black default, although irrelevant with E501 ignored
+[tool.black]
 line-length = 88
```

Comment:
This is still adding configuration for black and possibly changing the configuration of ruff.

Reply:
I removed it.